Assets/Plugins/Source/Editor/Utility/UnityProjectUtility.cs

```diff
@@ -77,16 +79,18 @@ public static bool CanFindMissingScriptsInPrefabs()
         }
 
         [MenuItem("Tools/Find Missing Scripts/In Scenes")]
-        private static void FindMissingScripts()
+        private static void FindMissingScriptsInScenes()
         {
             // Store the currently open scene
             var previouslyOpenScene = SceneManager.GetActiveScene().path;
 
+            IList<(string message, Object context)> missingScriptMessages = new List<(string message, Object context)>();
+
             // Iterate over all the scenes
             foreach (string scenePath in EditorBuildSettingsScene.GetActiveSceneList(EditorBuildSettings.scenes))
             {
                 // Open the scene
-                EditorSceneManager.OpenScene(scenePath);
+                Scene currentScene = EditorSceneManager.OpenScene(scenePath, OpenSceneMode.Single);
 
                 // Get all the game objects
                 GameObject[] gameObjects = Object.FindObjectsOfType<GameObject>();
```

Comment:
If an object is inactive in the scene, will this find it? Should we pass in "includeInactive: true" to this function?

Reply:
Good catch - I didn't realize that it defaulted to exclude inactive objects. I'll change that.

Reply:
Fixed.